runtime/lua/vim/_defaults.lua

```diff
@@ -270,17 +286,20 @@ do
           if rr and gg and bb then
             local luminance = (0.299 * rr) + (0.587 * gg) + (0.114 * bb)
             local bg = luminance < 0.5 and 'dark' or 'light'
-            if bg ~= vim.o.background then
-              vim.o.background = bg
-            end
+            setbg(bg)
           end
 
           return true
         end
       end,
     })
 
-    io.stdout:write('\027]11;?\027\\')
+    local query = '\027]11;?\027\\'
+    if os.getenv('TMUX') then
+      query = string.format('\027Ptmux;\027%s\027\\', query)
+    end
```

Comment:
Actually I do not think we should do this. Passthrough is disabled in tmux by default, and tmux now responds to OSC 11 requests "natively" on newer versions. And the original implementation of this did not use the passthrough sequence either.